pkg/fileutil/path_test.go

```diff
@@ -0,0 +1,52 @@
+package fileutil_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ddev/ddev/pkg/fileutil"
+	"github.com/ddev/ddev/pkg/testcommon"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestRemoveAllExcept tests copying a directory.
+func TestRemoveAllExcept(t *testing.T) {
+	testDir, _ := os.Getwd()
+	assert := assert.New(t)
+	require := require.New(t)
+
+	sourceDir := filepath.Join(testDir, "testdata", "remove_all_except")
+	targetBaseDir := testcommon.CreateTmpDir("TestRemoveAllExcept")
+	targetDir := filepath.Join(targetBaseDir, "testdata")
+	defer os.RemoveAll(targetBaseDir)
```

Comment:
Much preferred here is t.Cleanup(), and catch and assert on the result of RemoveAll, so we know if there's been a problem.